src/utils/cli-parser.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Parse command line
+ * @param {string[]} args arguments
+ * @returns {object|null} options object or null if the user needs help
+ */
+function parseCLI(args) {
```

Comment:
Any particular reason you didn’t use a small utility lib like [commander](https://www.npmjs.com/package/commander) to handle your command-line interface? That would save you the trouble of parsing the input by hand (and testing it).

Reply:
I was just unsure about which one to use because there are so many choices.